### PR TITLE
feat(remoteConfig): add Firebase SDK

### PR DESCRIPTION
**Pull request by dkremniov-bc**
## Description (optional)
Adds Firebase Remote Config implementation using [Load new values for next startup](https://firebase.google.com/docs/remote-config/loading#strategy_3_load_new_values_for_next_startup) loading strategy. Should be a superior replacement for all existing feature flag management solutions.

This initial version supports only Boolean values.

## Testing Steps (optional)
Set `web_ff_super_app` to true in Firebase console then check Redux store for the value in `remoteConfig`. Be aware of [the SDK throttling](https://firebase.google.com/docs/remote-config/get-started?platform=web#throttling). In case if SDK request fails, you will still see default config in the Redux; however, instead of dispatching `remoteConfig/setConfig` - `remoteConfig/initializeFailed` will be dispatched.

### CODE REVIEW

**Comment by alokeutpal** [comment]
Wow nice game keep it up

**Review by TheLeoB — DISMISSED** [review]

File: packages/blockchain-wallet-v4-frontend/webpackBuilder.js [hunk 1]
```diff
@@ -239,6 +257,8 @@ const buildDevServerConfig = (
           'connect-src',
           "'self'",
           'data:',
+          `https://firebaseinstallations.googleapis.com/v1/projects/${process.env.FIREBASE_PROJECT_ID}`,
+          `https://firebaseremoteconfig.googleapis.com/v1/projects/${process.env.FIREBASE_PROJECT_ID}`,
```
[the remark this answers is not in the document]
Comment (dkremniov-bc) [reply packages/blockchain-wallet-v4-frontend/webpackBuilder.js:261]: Does [this PR](https://github.com/blockchain/ci-pipelines/pull/601) fully covers that?

File: packages/blockchain-wallet-v4-frontend/src/scenes/app.tsx [hunk 2]
```diff
@@ -117,170 +118,172 @@ const App = ({
...
+                          </Suspense>
+                        </ConnectedRouter>
+                        <SiftScience userId={userData.id} />
+                      </UrqlProvider>
+                    </MediaContextProvider>
+                  </PersistGate>
+                </TranslationsProvider>
+              </ThemeProvider>
+            </ConstellationTP>
+          </RemoteConfigProvider>
```
[the remark this answers is not in the document]
Comment (dkremniov-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/app.tsx:286]: Hopefully, will be refactored as a part of the superapp web work.

File: packages/blockchain-wallet-v4-frontend/webpackBuilder.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (alokeutpal) [reply packages/blockchain-wallet-v4-frontend/webpackBuilder.js:261]: Doing so much while people have benefited at all - ?

[the remark this answers is not in the document]
Comment (alokeutpal) [reply packages/blockchain-wallet-v4-frontend/webpackBuilder.js:261]: Doing so much while people have benefited at all - ?

**Pull request merged**